.gitignore

```diff
@@ -0,0 +1,6 @@
+*.log
+target/
```

Comment:
It would be probably better to put all this filters except of `target/` to `.git/info/exclude` file. The reason is that each developer is free to choose any editor/IDE which can store different configuration files, not only `.project` or similar. So, it's up to developer to filter out such files. Thus, `.gitignore` will contain only directories/files generated during compilation.